src/app/organizer/table-types.ts

```diff
@@ -52,6 +54,27 @@ export interface ColumnDefinition<V extends Value = Value> {
    * but sometimes a custom stat should be limited to only displaying for a certain class
    */
   limitToClass?: DestinyClass;
+
+  /**
+   * A name for this column when it is output as CSV. This will reuse the value
+   * function as-is. We could reuse the header, but that's localized, while
+   * historically our CSV column names haven't been. I tried to combine this and
+   * `csvVal` but ran into type covariance issues.
```

Comment:
Basically, if you do:
```ts
  csv: string | ((value: V, item: DimItem, spreadsheetContext: SpreadsheetContext) => CSVColumn | CSVColumn[]);
```

Then TypeScript will complain that, for example, `ColumnDefinition<string>` can't assign a `((value: string, item: DimItem, spreadsheetContext: SpreadsheetContext) => CSVColumn | CSVColumn[])` to `((value: Value, item: DimItem, spreadsheetContext: SpreadsheetContext) => CSVColumn | CSVColumn[])`

Reply:
You can use the good old bivariance hack for this if you really want to combine them:
```
type CSVOutput<V> = {
  bivarianceHack(
    value: V,
    item: DimItem,
    spreadsheetContext: SpreadsheetContext,
  ): CSVColumn | CSVColumn[];
}['bivarianceHack'];
```
then `csv?: string | CSVOutput<V>;` works

Reply:
Aha! I knew there was a hack like this, but couldn't remember it.